tests/avocado/avocado_qemu/test.py

```diff
@@ -0,0 +1,30 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (C) 2017 Red Hat Inc
+#
+# Authors:
+#  Amador Pahim <apahim@redhat.com>
+
+
+from avocado import Test
+from .base import VM
+
+
+class QemuTest(Test):
```

Comment:
Well from `qemu` point of view the `QemuTest` is `base`. I don't see a big reason for splitting those files in two at this point.